en/sim_gazebo_gz/gazebo_vehicles.md

```diff
@@ -0,0 +1,62 @@
+# Gazebo Vehicles
+
+This topic lists/displays the vehicles supported by the PX4 [Gazebo](../sim_gazebo_gz/README.md) simulation and the `make` commands required to run them (the commands are run from a terminal in the **PX4-Autopilot** directory).
+
+Supported vehicle types include: mutirotor, VTOL, Plane.
+
+:::note
+The [Gazebo](../sim_gazebo_gz/README.md) page shows how to install Gazebo.
+:::
+
+## Multicopter
+<a id="x500-quadrotor"></a>
+### x500 quadrotor
```

Comment:
Note, headings use first letter capitalization of all words (except maybe 'a', 'of' etc). You only need an anchor if you can't use the default on.

```suggestion
## Multicopter

### X500 Quadrotor
```